src/dbnode/storage/database_bootstrapped_test.go

```diff
@@ -34,16 +35,17 @@ func TestDatabaseIsBootstrappedAndDurable(t *testing.T) {
 
 	var (
 		validIsBootstrapped                  = true
-		validShardSetAssignedAt              = time.Now()
-		validLastBootstrapCompletionTime     = validShardSetAssignedAt.Add(time.Second)
-		validLastSuccessfulSnapshotStartTime = validLastBootstrapCompletionTime.Add(time.Second)
+		validShardSetAssignedAt              = xtime.ToUnixNano(time.Now())
+		validLastBootstrapCompletionTime     = xtime.ToUnixNano(validShardSetAssignedAt.ToTime().Add(time.Second))
+		validLastSuccessfulSnapshotStartTime = xtime.ToUnixNano(validLastBootstrapCompletionTime.ToTime().Add(time.Second))
+		zeroTime                             = xtime.UnixNano(0)
```

Comment:
nit: This can just be `zeroTime xtime.UnixNano` and will get the zero value by default.